stable/rabbitmq/templates/svc.yaml

```diff
@@ -16,6 +16,9 @@ spec:
   - name: amqp
     port: {{ .Values.rabbitmq.nodePort }}
     targetPort: amqp
+    {{- if (and (eq .Values.service.type "NodePort") (not (empty .Values.rabbitmq.nodePort))) }}
```

Comment:
According to the [failing test](https://circleci.com/gh/kubernetes/charts/8782?utm_campaign=vcs-integration-link&utm_medium=referral&utm_source=github-build-link):
```
[ERROR] templates/: render error in "rabbitmq/templates/svc.yaml": template: rabbitmq/templates/svc.yaml:19:27: executing "rabbitmq/templates/svc.yaml" at <.Values.service.type>: can't evaluate field type in type interface {}
```

You should use
```
.Values.serviceType "NodePort"
```
instead of
```
.Values.service.type "NodePort"
```